src/templates/handlerUtils.ts

```diff
@@ -63,3 +67,110 @@ export const getMultiValueHeaders = (
   }
   return multiValueHeaders
 }
+
+export const augmentFsModule = ({
+  promises,
+  staticManifest,
+  pageRoot,
+  getBase,
+}: {
+  promises: typeof fs.promises
```

Comment:
[dust] You can also import `promises` from `fs and use `typeof promises`

Huh I would have thought there was a type you could refer to directly but doesnt look like it

Reply:
ikr. weird

Reply:
I didn't do that, because I'm using `promises` there and don't like shadowing imports